test/kata4.humanCatDogYears.test.js

```diff
@@ -1,3 +1,7 @@
 const { humanCatDogYears } = require("../src");
 
-// Look Ma, no handlebars!!!
+describe("humanCatDogYears", () => {
+  it("returns array of human, cat and dog years when passed human year", () => {
```

Comment:
Could more tests be added here again? At the least all the code paths in the original code should be accounted for (1, 2, more than 2). What about if a negative number or a float is provided?